src/object/schema_manager.c

```diff
@@ -14785,7 +14786,16 @@ sm_add_constraint (MOP classop, DB_CONSTRAINT_TYPE constraint_type, const char *
 	      goto error_exit;
 	    }
 	}
-
+      else if (!use_stat_estimation)
+	{
+	  /* if NOT use_stat_estimation then stats is not updated at update_class() */
+	  /* this routine is for create index statement */
+	  error = sm_update_statistics (newmop, STATS_WITH_SAMPLING);
```

Comment:
if the statistics are changed just only when an index is created, what happen for updating, inserting, or deleting?

Reply:
use_stat_estimation=no
statistics are not changed when updating, inserting, deleting.
Only when executing 'update statistics' stmt, 'create index' stmt and 'drop index' stmt, the statistics are changed.